.github/workflows/docs.yml

```diff
@@ -19,9 +19,9 @@ jobs:
       - name: git checkout
         uses: actions/checkout@v4
       - name: Install dependencies
-        run: pip install sphinx-design sphinxawesome-theme rstfmt
+        run: pip install -r docs/requirements.txt
```

Comment:
We're assuming the user is in the docs directory.

Reply:
Are we? https://github.com/php/php-src/actions/runs/11872317688/job/33085877287#step:3:2 

Therefore, if so, then IMHO we should stop assuming and go with the reality that the CI build user is *not* (and never was) i the docs directory and CI run builds in-tree at work-tree root.